pyvcloud/vcd/vapp.py

```diff
@@ -173,3 +174,36 @@ def connect_vm(self, mode='DHCP', reset_mac_address=False):
                 EntityType.NETWORK_CONNECTION_SECTION.value,
                 self.resource.Children.Vm[0].NetworkConnectionSection
                 )
+
+    def attach_disk_to_vm(self, disk_href, disk_type, disk_name, vm_name):
```

Comment:
Add docstrings for any new methods added. You can follow docstring formats in pyvcloud/vdc.py

Reply:
done.